src/features/supervisor-app/hooks/supervisor-app.ts

```diff
@@ -1,12 +1,13 @@
 import * as semver from 'balena-semver';
 import * as _ from 'lodash';
-
+import { createReleaseServiceInstalls } from '../../ci-cd/hooks/service-installs';
```

Comment:
This is a big red flag to me that you're importing a function from the hooks of another feature and makes me think that it should be done there instead. And checking the code I see no reason why the behavior can't be handled by adding a hook there that follows the existing
```ts
			if (request.values.should_be_running__release != null) {
				await createReleaseServiceInstalls(api, affectedIds, {
					id: request.values.should_be_running__release,
				});
			}
```
style but instead does
```ts
			if (request.values.should_be_managed_by__release!= null) {
				await createReleaseServiceInstalls(api, affectedIds, {
					id: request.values.should_be_managed_by__release,
				});
			}
```
ie same behavior with the different key

Reply:
Resolving because these changes were rolled back.